content/packages/index.md

```diff
@@ -131,9 +131,7 @@ In addition to [purrr](https://purrr.tidyverse.org), which provides very consist
 
 ## Model
 
-Modelling within the tidyverse is largely a work in progress. You can see some of the pieces in the [recipes](https://github.com/topepo/recipes) and [rsample](https://github.com/topepo/rsample) packages but we do not yet have a cohesive system that solves a wide range of challenges. This work will largely replace the [modelr](https://github.com/tidyverse/modelr) package used in [R4DS](https://r4ds.had.co.nz/).
-
-You may also find [broom](https://github.com/tidyverse/broom) to be useful: it turns models into tidy data which you can then wrangle and visualise using the tools you already know.
+Modelling with the tidyverse is based on the collection of [tidymodels packages](https://www.tidymodels.org/), which largely replace the [modelr](https://github.com/tidyverse/modelr) package used in [R4DS](https://r4ds.had.co.nz/). These packages provide a comprehensive foundation for creating and using models of all types. Visit the [_Getting Started_](https://www.tidymodels.org/start/) guide or, if you already have a background in modelling, go straight to the [_Learn_](https://www.tidymodels.org/learn/) page.  
```

Comment:
```suggestion
Modeling with the tidyverse uses the collection of [tidymodels packages](https://www.tidymodels.org/), which largely replace the [modelr](https://github.com/tidyverse/modelr) package used in [R4DS](https://r4ds.had.co.nz/). These packages provide a comprehensive foundation for creating and using models of all types. Visit the [_Getting Started_](https://www.tidymodels.org/start/) guide or, for more detailed examples, go straight to the [_Learn_](https://www.tidymodels.org/learn/) page.  
```

I'm suggesting this change because `Getting Started` is going to be helpful for experienced modeling folks who don't yet have experience with tidymodels.

Reply:
For consistency, "Getting Started" --> "Get started" (then matches top navbar)